go.mod

```diff
@@ -386,6 +386,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require github.com/google/go-querystring v1.1.0
```

Comment:
Can you move this into the require block above?
Go mod creates new require blocks by default

Reply:
Done